changelog_unreleased/handlebars/10586.md

```diff
@@ -0,0 +1,40 @@
+#### Do not break opening mustache and path (#10586 by @dcyriller)
+
+<!-- prettier-ignore -->
+```handlebars
+{{!-- Input --}}
+<GlimmerComponent
+  @errors={{or this.aVeryLongProperty (and this.aProperty (v-get bike "number" "message"))}}
+  data-test-beneficiary-account-number
+/>
+<GlimmerComponent
+  @progress={{aPropertyEngdingAfterEightiethColumnToHighlightAWeirdClosingParenIssue}}
+/>
+
+{{!-- Prettier stable --}}
+<GlimmerComponent
+  @errors={{
+    or this.aLongProperty (and this.aProperty (v-get bike 'number' 'message'))
+    this.aLongProperty
+    (and this.aProperty (v-get bike 'number' 'message'))
```

Comment:
This doesn't match the input. Please fix.

Reply:
sorry